pandas/tests/scalar/test_na_scalar.py

```diff
@@ -69,6 +72,25 @@ def test_comparison_ops():
         assert (other <= NA) is NA
 
 
+@pytest.mark.parametrize(
+    "value", [0, 0.0, False, np.bool_(False), np.int_(0), np.float_(0)]
+)
+def test_pow_special(value):
+    result = pd.NA ** value
+    assert isinstance(result, type(value))
+    assert result == 0
+
+
+@pytest.mark.parametrize(
+    "value", [1, 1.0, True, np.bool_(True), np.int_(1), np.float_(1)]
+)
+def test_rpow_special(value):
+    result = value ** pd.NA
+    assert result == 1
+    if not isinstance(value, (np.float_, np.bool_, np.int_)):
```

Comment:
Somebody (Cython?) is converting these to Python scalars.

Reply:
have you tried with/without numexpr?

Reply:
How can numexpr be involved in this?

Reply:
i dont know, but if we're brainstorming things that can do surprising conversions, it comes to minid